langchain/document_loaders/telegram.py

```diff
@@ -82,24 +82,26 @@ class TelegramChatApiLoader(BaseLoader):
 
     def __init__(
         self,
-        chat_url: Optional[str] = None,
+        chat_entity: Optional[str] = None,
```

Comment:
@0xRaduan thanks for the contribution. Could you explain why the change of chat_url to chat_entity?

Reply:
Yep, definitely, @eyurtsev.

Per the telethon package, iter_messages(which is used to perform API call to Telegram, to retrieve chat history, [is marked as `Entity`](https://docs.telethon.dev/en/stable/modules/client.html#telethon.client.messages.MessageMethods.iter_messages))

I wanted to add a proper type, but it required me to run poetry lock(so I can get intellisense in the IDE). If you want me, I can do that as well.

Reply:
technically a breaking change, but probably ok given this interface is brand new